README.md

```diff
@@ -49,6 +49,102 @@ To start using the KPPTR-firmware and exploring its capabilities, follow these s
 
 Please check [ESP-IDF docs](https://docs.espressif.com/projects/esp-idf/en/latest/get-started/index.html) for getting started instructions.
 
+## Hardware
+### Prototype PCB
+Hardware fot KPPTR is developed in repository [PTR_tracker_hardware](https://github.com/PTR-projects/PTR_tracker_hardware). 
+As of Dec 2023 boards are not in serial production yet. Prototype design is almost fully tested, but still, it's a prototype. 
+
+### Bread board
+Not all development require dedicated PCB. As we use standard ESP32-S3 processor, development boards could be used. 
+
+#### ESP32-S3-WROOM-1-N16R8
+One of them is `ESP32-S3-WROOM-1-N16R8`. It can be flashed directly via USB cable without additional programmer. Only two
+adjustments needs to be done. Run in `PTR_mega_firmware` directory command `idf.py menuconfig` and using menu:
+1. Set partition table definition to match 16MB flash: `Serial flasher config -> flash size -> 16MB`
+2. Set flash size to 16mb in project settings: `Partition table -> custom partition CSV file -> partitions-16mb.csv` 
+3. Quit menu saving changes
+4. Rebuild project
+
+#### Flashing firmware
+1. Attach ESP32 board with USB cable to computer. `pwr` led should turn on.
+2. Holding `rst` button press once `boot` button. This will initiate programming sequence on ESP32 side.
+2. Find how your system recognise USB attached ESP32 board with command
+    ```bash
+    $ ls -l /dev/cu.*
```

Comment:
Serial port naming "cu.*" works only on OSX. Can you add variant for Linux and Windows? Windows command will be tricky.

Reply:
I don't have machine with Windows. Added MacOS and Linux.